src/components/data-table.jsx

```diff
@@ -69,6 +69,7 @@ const DataTableHead = ({ selectable, columns, onHeaderClick }) => (
             key={name}
             className={className}
             onClick={onClick}
+            data-testid={sortable ? 'sortable-header' : undefined}
```

Comment:
Just wondering why not just call it something like `header-cell` rather than conditioning?

Reply:
Good point, I guess I was just trying to replicate the previous specific class selector and ended up still being over-specific. I replaced it now in the test and removed the requirement for a testid